src/js/tabs/firmware_flasher.js

```
@@ -1472,6 +1567,13 @@ firmware_flasher.initialize = async function (callback) {
             $("a.exit_dfu").removeClass("disabled");
         }
 
+
+            const isFlashOnConnect = $("input.flash_on_connect").is(":checked");
```

ESLint

[error] Expected indentation of 8 spaces but found 12.

(indent)